src/ServiceInsight/ValueConverters/SearchStringConverter.cs

```diff
@@ -0,0 +1,25 @@
+﻿namespace ServiceInsight.ValueConverters
+{
+    using System;
+    using System.Globalization;
+    using System.Windows.Data;
+
+    public class SearchStringConverter : IValueConverter
+    {
+        public object Convert(object value, Type targetType, object parameter, CultureInfo culture)
+        {
+            var str = string.Format("{0}", value);
+            if (str.StartsWith("\"") && str.EndsWith("\""))
+            {
+                str = str.Substring(1, str.Length - 2);
+            }
+
+            return str;
+        }
+
+        public object ConvertBack(object value, Type targetType, object parameter, CultureInfo culture)
+        {
+            return string.Format("\"{0}\"", value);
+        }
```

Comment:
This could be more for my own education than anything else, but looking purely at the method signatures, I would naively write these methods like so:

```C#
public object Convert(object value, Type targetType, object parameter, CultureInfo culture)
{
    if (targetType != typeof(string))
    {
        throw new InvalidOperationException($"Cannot convert value to target type '{targetType}'.");
    }

    if (value == null)
    {
        return null;
    }

    var stringValue = value as string;

    if (stringValue == null)
    {
        throw new InvalidOperationException($"The value is not a string.");
    }

    return stringValue.StartsWith("\"", true, culture) && stringValue.EndsWith("\"", true, culture)
        ? stringValue.Substring(1, stringValue.Length - 2)
        : stringValue;
}

public object ConvertBack(object value, Type targetType, object parameter, CultureInfo culture) =>
    string.Format(culture, "\"{0}\"", value);
```

However, I don't know what assumptions can be made wrt the WPF execution context, and what does or does not need to be done. Things to note about the above snippet:

- Respecting the `targetType` argument and throwing if it's not a `string`
- Short circuiting the case where `value` is `null`.
- Avoiding the string allocation by safe casting `value` to a `string`.
- Throwing if value is not a `string`
- using the supplied `culture` argument where appropriate (in both methods)